e2e/test/prerequisites/E2ETestsSetup/test-resources.bicep

```diff
@@ -100,7 +98,7 @@ resource storageAccount 'Microsoft.Storage/storageAccounts@2021-02-01' = {
     name: 'Standard_LRS'
     tier: 'Standard'
   }
-  kind: 'Storage'
```

Comment:
I think someone edited the json by hand and didn't update the bicep file, so this fixes that.